contracts/src/subnetregistry/RegisterSubnetFacet.sol

```diff
@@ -57,6 +57,18 @@ contract RegisterSubnetFacet is ReentrancyGuard {
             functionSelectors: s.subnetActorPauserSelectors
         });
 
+        diamondCut[5] = IDiamond.FacetCut({
+            facetAddress: s.SUBNET_ACTOR_DIAMOND_CUT_FACET,
```

Comment:
Is it reasonable to add OwnershipFacet here? We need owner transferring

Reply:
perfect :heavy_check_mark: